inductiva/benchmarks/benchmark.py

```diff
@@ -243,3 +245,32 @@ def filter_distinct_columns(info):
         return filter_distinct_columns(info) \
             if columns == ColumnExportMode.DISTINCT \
             else info
+
+    def terminate(self) -> Self:
```

Comment:
I would prefer to have this method implemented in the `Project` class.

Reply:
It could be implemented there, but I think it should be implemented in the ```Benchmark``` class for now, as it is more directly related to machine groups, whereas the ```Project``` class is more related to tasks. Wdyt?

I also think there are other methods / features in the ```Benchmark``` class that could be moved to the base class, but this could be done at a later stage during refactoring.